src/lemlib/chassis/differential.cpp

```diff
@@ -72,12 +71,12 @@ Differential::Differential(Drivetrain drivetrain, ControllerSettings linearSetti
  */
 void Differential::initialize() {
     // calibrate odom
-    odom->calibrate();
+    this->odom->calibrate();
```

Comment:
why are we using `this->` when accessing class members?

Reply:
You should always use this-> for class members to differentiate things from in/outside of the class. It helps ppl to understand the code and allow you to have the function parameters named the same as the class members.